Source/ThirdParty/libwebrtc/Source/webrtc/pc/legacy_stats_collector.cc

```diff
@@ -188,9 +202,11 @@ void ExtractStats(const cricket::VoiceReceiverInfo& info,
       {StatsReport::kStatsValueNameAccelerateRate, info.accelerate_rate},
       {StatsReport::kStatsValueNamePreemptiveExpandRate,
        info.preemptive_expand_rate},
+RTC_PUSH_IGNORING_NARROWING_CONST_REFERENCE()
       {StatsReport::kStatsValueNameTotalAudioEnergy, static_cast<float>(info.total_output_energy)},
       {StatsReport::kStatsValueNameTotalSamplesDuration,
        static_cast<float>(info.total_output_duration)}};
+RTC_POP_IGNORING_NARROWING_CONST_REFERENCE()
```

Comment:
These have already been fixed using `static_cast<float>()`.

See this command for what it looked like before:

`git show WebKit-7619.1.7.3:Source/ThirdParty/libwebrtc/Source/webrtc/pc/legacy_stats_collector.cc`